source/leuven/genius_2_rocky.rst

```diff
@@ -0,0 +1,116 @@
+   for the CPU nodes (which are inside the ``batch`` partition), and::
+
+      module load cluster/genius/gpu_p100
+     
+   for the GPU node (which is in the ``gpu_p100`` partition).
+   This will set your module path to point to the new modules for Rocky 8.
+
+
+.. _impact_on_central_software:
+
+Impact on centrally installed modules
+-------------------------------------
+Current Genius software needs to be reinstalled because dependencies on the OS libraries will break the existing software.
+All centrally installed modules which have seen recent use, starting from 2018a toolchains, will be re-installed (as far as possible). Older toolchains will not be migrated.
+Unused modules are not migrated automatically, but we strongly advise to use modules in the most recent toolchains (2021a).
+
+In order to distinguish between the old CentOS 7 and the new Rocky 8 software installations, an additional level is introduced in the ``/apps`` hierarchy::
+
+   /apps/leuven/${VSC_OS_LOCAL}/${VSC_ARCH_LOCAL}${VSC_ARCH_SUFFIX}/TOOLCHAIN_VERSION/modules/all
+
+e.g.::
+
+   /apps/leuven/rocky8/skylake/2018a/modules/all 
+  
+This convention is in line with other VSC sites and will also be used on wICE and future clusters.
+
+
+.. _impact_on_user_installed_software:
+
+Impact on user-installed software
+---------------------------------
+If you have installed a software package yourself in your own account, and you did this on a Genius CentOS 7 node, it must be recompiled on Genius on a node with the new OS.
+This can be done on one of the available test nodes. Please request access to the lpt2_rocky8_pilot group.
+
+.. _impact_on_conda:
+
+Impact on Conda environments
+----------------------------
+The Conda environment you installed might need reinstallations. If you already have a Conda environment that works on wICE, it also should work on Genius after the migration.
+If you only have a Conda environment working on Genius, it's best to create a new Conda installation. In this case, it is recommended to recreate your environment for full compatibility with the new OS. Best practice is to choose a new installation folder with explicit mention of the new OS, e.g. ::
+
+   ${VSC_DATA}/miniconda3-rocky
+  
+You can then use this Conda environment after the migration. You can prepare this on the test nodes.
+
+
+.. _impact_on_starting_jobs:
+
+Impact on starting jobs on Genius and wICE
+------------------------------------------
+In order to minimize the changes you need to make to your jobscripts, an appropriate module path (``$MODULEPATH``) will be set by default at the start of your job of the the migration of Genius to Rocky 8 OS. This new module path will now contain all toolchain versions, starting from 2018a on Genius and starting from 2021a on wICE. This is an important change! Previously the modulepath was set to a single toolchain, that would not change over time. You might have set a module path in your jobscripts to refer to newer toolchains.
```

Comment:
maybe add excplicitely
... all toolchain versions, so all the modules, starting from...

Module use and module unuse commands will not be necessary to see the most recent versions of the mdules.